src/Builder.hs

```diff
@@ -95,10 +96,13 @@ data Builder = Alex
              | Objdump
              | Patch
              | Perl
+             | Python
              | Ranlib
+             | RunTest
              | Sphinx SphinxMode
              | Tar TarMode
              | Unlit
+             | Validate
```

Comment:
This seems to be irrelevant for this particular PR? I suggest to drop this to avoid confusion.